GitUpKit/Views/GIStashListViewController.m

```diff
@@ -307,6 +307,7 @@ - (IBAction)dropStash:(id)sender {
       [self.undoManager setActionName:NSLocalizedString(@"Drop Stash", nil)];
       [[self.undoManager prepareWithInvocationTarget:self] _undoDropStashWithPreviousState:currentState ignore:NO];  // TODO: We should really use the built-in undo mechanism from GCLiveRepository
       [self.repository notifyRepositoryChanged];
+      [_tableView selectRowIndexes:[NSIndexSet indexSetWithIndex:MAX(0, row - 1)] byExtendingSelection:NO];
```

Comment:
I would prefer this which explicitly handles the edge cases vs relying on "patching" them afterwards with `MAX()` or relaying on `-selectRowIndexes: byExtendingSelection:` being a no-op if the table is empty (I'm assuming here that `NSTableView` keeps the 0th row selected if deleting row 0):
```
if (row > 0) {
  [_tableView selectRowIndexes:[NSIndexSet indexSetWithIndex:(row - 1) byExtendingSelection:NO];
}
```